src/AppInstallerCLIE2ETests/TestData/Manifests/TestZipInstaller_Portable.2.0.0.0.yaml

```diff
@@ -0,0 +1,18 @@
+PackageIdentifier: AppInstallerTest.TestZipInstallerWithPortable
+PackageVersion: 2.0.0.0
+PackageName: TestZipInstallerWithPortable
+PackageLocale: en-US
+Publisher: AppInstallerTest
+License: Test
+ShortDescription: E2E test for installing a zip with portable.
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://localhost:5001/TestKit/AppInstallerTestZipInstaller/AppInstallerTestZipInstaller.zip
+    InstallerType: zip
+    InstallerSha256: <ZIPHASH>
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: AppInstallerTestExeInstaller.exe

```

Comment:
There should be some tests which consider two entries with the same `RelativeFilePath`, Two entries with the same `PortableCommandAlias`, and possibly Exact duplicate entries?

I was doing some testing and I don't think that these cases are being validated against at all. (#2523) 
Perhaps @denelon can provide his thoughts on if these validations would be required